flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/calcite/LogicalSink.scala

```diff
@@ -37,11 +37,12 @@ final class LogicalSink(
     traitSet: RelTraitSet,
     input: RelNode,
     sink: TableSink[_],
-    sinkName: String)
+    sinkName: String,
+    val staticPartitions: Map[String, String])
```

Comment:
I think such information should be carried by `PartitionableTableSink`, but not this `LogicalSink`

Reply:
So we need add `getStaticPartitions` to `PartitionableTableSink`?

Reply:
Guess you're right, it's better to transfer the information through `LogicalSink`. We already add such information to `CatalogSinkModifyOperation`

Reply:
Yes, actually the sink concept from `CatalogSinkModifyOperation` is `insert into ....` in SQL. So I think Sink contains static partitions information is not strange.